applications/CompressiblePotentialFlowApplication/custom_elements/incompressible_potential_flow_element.h

```diff
@@ -23,23 +23,22 @@
 
 namespace Kratos
 {
+template <unsigned int TNumNodes, unsigned int TDim>
+struct ElementalData
```

Comment:
Just curious, what is the difference in defining the struct here or inside the class?

Reply:
from outside the one in the class is to be used as `IncompressiblePotentialFlowElement::ElementalData` if defined in the public section or not accessible if defined in the private section

Reply:
My idea is to define the struct only in the utilities at the end, since once all of the functions are migrated it will be only used there. Also right now it is defined both in the incompressible and the compressible element.